astro/src/content/blog/securely-implement-oauth-in-react.mdx

```diff
@@ -30,7 +30,7 @@ In addition to React, we'll use a NodeJS backend. One major benefit of using a b
 
 <Aside type="tip" nodark="true">This blog post has been [superseded by a newer React/OAuth tutorial](/blog/2021/11/11/how-to-authenticate-your-react-app).</Aside>
 
-If you're in-the-know on OAuth, you're probably aware that some people use PKCE or the rightfully deprecated Implicit Flow to get around the Client Secret constraint. We're sticking to Authorization Code Flow, because setting up a server is the most secure solution for the majority of cases and quite simple once you know how. Don't know what any of that means? Don't worry about it — you don't need to know anything about OAuth to follow this example. 
```

Comment:
Are all of these the result of two conflicting editor settings? Someone with trailing spaces by default and yours removes them? If so we should figure out how to get that consistent.

Reply:
I don't use intellij, I use vim and sometimes have whitespace on a line.

Other folks use other editors.

Is this something we can automate with a GH script?

Agree that this doesn't seem to be good to wrap into this PR, but not sure that's a blocker.

Reply:
Probably - IJ is configured to remove trailing white space. I would prefer that be the default on any editor we use to edit docs.